scripts/fast-start.sh

```diff
@@ -19,8 +19,7 @@ IP="$(which ip)"
 APPSCALE_CMD="$(which appscale)"
 APPSCALE_UPLOAD="$(which appscale-upload-app)"
 GOOGLE_METADATA="http://169.254.169.254/computeMetadata/v1/instance/"
-GUESTBOOK_URL="http://www.appscale.com/wp-content/uploads/2014/07/guestbook.tar.gz"
-GUESTBOOK_APP="/root/guestbook.tar.gz"
+GUESTBOOK_APP="/root/appscale/AppServer/demos/appengine-guestbook-python"
```

Comment:
Can we have a $APPSCALE_HOME variable here instead of /root/appscale? 
I believe it's already available in an AppScale environment.